src/components/AboutUs/AboutUs.jsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+import CloseSharpIcon from '@mui/icons-material/CloseSharp';
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 600,
+  bgcolor: 'background.paper',
+  borderRadius: '10px !important',
+  boxShadow: 24,
+  p: 4,
+  color:'black',
+};
```

Comment:
Does this have to be outside of the function?

Reply:
Default on mui modal example them define an outside function component, but in best practice, I don't know whether outside or inside are best.

Reply:
Ok. I'll resolve this for now. This is a minor style-related issue. We can refactor later.

Reply:
Ok,Thanks krub P'Khong